src/Bundle/Doctrine/ORM/CreatePaginatorTrait.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\Doctrine\ORM;
+
+use Doctrine\ORM\EntityRepository as DoctrineEntityRepository;
+use Doctrine\ORM\QueryBuilder;
+use Pagerfanta\Adapter\ArrayAdapter;
+use Pagerfanta\Doctrine\ORM\QueryAdapter;
+use Pagerfanta\Pagerfanta;
```

Comment:
Are we ready for
```php
Pagerfanta\PagerfantaInterface
```
yet or it should be kept as it for BC?

Reply:
It seems to be ok so, let's go.
https://3v4l.org/O58qm